Project.toml

```
@@ -1,7 +1,7 @@
 name = "UTCDateTimes"
 uuid = "0f7cfa37-7abf-4834-b969-a8aa512401c2"
 authors = ["Invenia Technical Computing Corporation"]
-version = "1.6.0"
+version = "1.7.0"
```

```suggestion
version = "1.6.1"
```

Sorry if we aren't introducing deprecations and just dropping the export I think a patch release is still fine.